etrago/tools/extendable.py

```diff
@@ -323,6 +298,85 @@ def extendable(self, line_max):
     return network
 
 
+def line_max_abs(network, buses, line_max_abs= {'380': 1000, '220': 700, '110':500, 'dc':2000}):
```

Comment:
Just out of curiosity, would you recommend to use these values? They seem a bit small, a default 380kV line already has a capacity of 1790MW...

Reply:
Good point! No, those values are not recommendable. I actually have a function for that upper bound definition, which I will integrate here as well. Thanks for bringing this up!